docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
+
+A how to guide for accessing and running applications on a kubernetes cluster.
+
+### Basic Setup
+The kubernetes cluster is accessed via the ```kubectl`` command.  Full details of using the ```kubectl``` command are described [elsewhere](docs/cli.md).  Make sure that ```kubectl``` is in your path by running
+
+```sh
+which kubectl
+```
+
+If that command returns a path to the binary, you are ready to go.  If it doesn't, you need to update your PATH to include the appropriate directory from the Kubernetes directory.
+
+### Commands
+#### run
+```sh
+kubectl run <pod-name> --image=nginx [--replicas=N] [--export [--external]]
+```
+
+This translates into a call to create a replication controller and optionally, a call to create a service which exposes the pods created by that replication controller.
+
+
+#### describe
+```sh
+kubectl describe <name>
+```
+
+Describe attempts a get against the three resource types (Pods, Replication Controllers, Services) and returns matching resource(s) for the named resource.  It ignores 404 errors, unless all of the get requests return 404.
```

Comment:
Basically, I want ```kubectl describe foo``` to not return an error if ```foo``` is one of pod, service, replication controller.  It should only return an error if it can't find anything with that name.  This is mainly for simplicity. 

If the user specifies a fully spec'd name, then there is only one GET request, and if it 404s then we'd return an error.